src/modules/bar/barNPCs.ts

```diff
@@ -1523,3 +1652,102 @@ export function octoTrip() {
     })
   }
 }
+
+
+export let GrumpyElfDialog: Dialog[] = [
+  {
+    name: 'xmasplaza-quest',
+    text: `Hey friend!  We need your help!   A big storm hit Xmas Plaza last night and caused all sorts of problems`,
+    offsetY: 24,
+  },
+  {
+    text: `It is a logistics nightmare over there.  We need your help to get back on schedule for Christmas.`,//to meet our deadline.
+    triggeredByNext: () => {
+      grumpyElf.playAnimation(`Worried_Bell  `, true, 4)
+    },
+  },
+  { 
+    text: `Will you help?  I can send you there now.`,
+    offsetY: 24,
+    isQuestion: true,
+    buttons: [
+      { label: 'YES', goToDialog: 'yes-xplaza' },
+      { label: 'NO', goToDialog: 'no-xplaza' },
+    ],
+  },
+  {
+    name: 'yes-xplaza',
+    text: "Great! I am sending you there now.  When you get there talk to the elf in charge.",
+    offsetY: 18,
+    triggeredByNext: () => {
+      grumpyElf.playAnimation('Waving_bell', true, 4)//Greet_Bell
+      grumpyElf.endInteraction()
+
+      teleportTo(CoordsEnum.XmasPlazaCoords)
+    },
+    isEndOfDialog: true,
+  },
+  {
+    name: 'no-xplaza',
+    text: "Alright then, all good! If you change your mind, head to Xmas Plaza and talk to the elf in charge.",
+    offsetY: 18,
+    triggeredByNext: () => {
+      grumpyElf.playAnimation('Worried_Bell', true, 4)
+      grumpyElf.endInteraction()
+    },
+    isEndOfDialog: true,
+  },
+  //after got wearable
+  {
+    name: 'xmasplaza-done',
+    text:
+      "Hey friend!  I heard you were a big help at Xmas Plaza and we are back on schedule. ",
+    offsetY: 24,
+  },
+  {
+    text: `I hope you like the gift we gave you. You earned it.  I would be happy to explain what makes them so special in case you have not figured it out yet.`,
+    isQuestion: true,
+    buttons: [
+      { label: 'YES', goToDialog: 'yes-info' },
+      { label: 'NO', goToDialog: 'no-info' },
+    ],
+  },
+  //START YES INFO on vision glasses dialog
+  {
+    name: 'yes-info',
+    text: `The vision glasses are a special kind of wearable called a portable experience.  `,
```

Comment:
Let's rather call it a "smart wearable" rather than portable experience
Over several discussions we agreed this is a better name to use in outwards comms about it